README.md

```diff
@@ -23,15 +23,18 @@ A CLI tool to import/export records to/from [kintone](https://www.kintone.com/).
    - Windows: `cli-kintone-win.zip`
    - Linux: `cli-kintone-linux.zip`
    - macOS: `cli-kintone-macos.zip`
-3. Extract executables from the ZIP file and run it.
+3. Extract the downloaded zip file and move the `cli-kintone` file to the `/usr/local/bin`.
+4. Run the `cli-kintone` command and confirm that the command is available.
+
+Note: If the terminal responds with an error stating that the command could not be found, the PATH environment variables may need to be edited to use cli-kintone.
```

Comment:
Currently, we don't instruct to place cli-kintone file to directory registered in `$PATH`.
There are several reasons.

- It is an optional step to be able to execute cli-kintone from anywhere and not a required step
- The `$PATH` depends on the users' OS and settings.
- cli-kintone is mainly used on Windows. There is no default directory like `/usr/local/bin` on Windows.

Thus, we want to keep it as is without adding to `$PATH` step.
However, I thought it is nice to separate the extraction process and execution process as you edited.

```suggestion
3. Extract the downloaded zip file
4. Run the extracted `cli-kintone.exe` on command prompt (Windows), or run `cli-kintone` on your terminal (Linux/macOS).
```

Reply:
I agree with removing the /usr/local/bin directory, but I think we should leave the note.
```
Note: If the terminal responds with an error stating that the command could not be found, the PATH environment variables may need to be edited to use cli-kintone.
```

In my case, when executing [cli-kintone v1.3.2](https://github.com/kintone/cli-kintone/releases/tag/v1.3.2), I could not run `cli-kintone` on my terminal even though I was in the cli-kintone folder. I had to specify full path (ex: /Downloads/cli-kintone-for-mac/cli-kintone --version) or edit the environment path to be able to run it.

Reply:
Okay, I understand that we need to remove the `and move the cli-kintone file to the /usr/local/bin` bit from Step 3.

Then I would like to keep the Note and add to it. Something like:

```md
Note:
* If the terminal responds with an error stating that the command could not be found, the PATH environment variables may need to be edited to use cli-kintone.
* For Linux and macOS users, it is recommended to move the `cli-kintone` file to the `/usr/local/bin`.
```

Reply:
We decided:

- Add `./` on the fourth step of the installation
- Add a note to run cli-kintone on any directory